app/routes.py

```diff
@@ -1,2 +1,153 @@
-from flask import Blueprint
+from app import db
+from app.models.task import Task
+from app.models.goal import Goal
+from flask import request, Blueprint, make_response, jsonify
+import requests
+import os
 
+goals_bp = Blueprint("goals", __name__, url_prefix="/goals")
+tasks_bp = Blueprint("tasks", __name__, url_prefix="/tasks")
+
+
+####################################################################
+#                              GOALS ROUTES                        #
+####################################################################
+
+@goals_bp.route("", methods=["GET", "POST"])
+def handle_goals():
+    if request.method == "GET":
+        sort_query = request.args.get("sort")
+        if sort_query == "asc": 
+            # Automatically sorts by asc
+            goals = Goal.query.order_by(goal.title)
+        elif sort_query == "desc":
+            # SQLAlchemy for sort by desc
+            goals = Goal.query.order_by(goal.title.desc())
+        else:
+            goals = Goal.query.all()
+        goals_response = []
+        for goal in goals:
+            goal_formatted = {
+                "id": goal.goal_id,
+                "title": goal.title
+            }
+            goals_response.append(goal_formatted)
```

Comment:
this works great! this is also a good opportunity to turn the dictionary formatting into a method similar to `goal_data_structure`. Perhaps we could bring both ideas together by returning the goal dictionary structure itself without the surrounding key "goal." That way you could use this method in two different scenarios.

this could also formatted the same way for `task_data_structure` method